### ISSUE TITLE
[a11y-defect-3]: Check the status of your appeal should be styled as an action link

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#63230]
## Point of contact

joshkimux

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

On the confirmation page of NoD, the user is given the option to check their status of their appeal. This is a link styled as a button that should be styled as a link.

<img width="702" alt="Screenshot 2023-08-07 at 2 25 38 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/f8f51fb2-b0d4-4d15-a6dd-b8aff54af171">

## Reproduction steps

1. Complete the NoD form
2. Right click on "check the status of your appeal" button
3. Confirm it is a link

## Proposed solution or next steps

As this is navigating to a URL, use a link. Given it's priority (as it is styled as a button), we recommend styling this as an action link.

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [VA Design System guidance on buttons](https://design.va.gov/components/buttons#guidance)

**Comment by eileen-coforma** [comment]
Confirmed the button is now a green action link.
